apps/api/Dockerfile

```diff
@@ -15,7 +15,9 @@ EXPOSE 8000
 # --------------------------------------------------------------------------
 
 # COMMAND TO RUN
-RUN   apt-get update \
+RUN  --mount=type=secret,id=github_token \
+  export GITHUB_PAT=`cat /run/secrets/github_token` \
```

Comment:
Read the secret file back into an env var.
* Yes, this means we went from an env var in the environment surrounding the build to a file and back to an env var inside the build. If I'm reading the docs right, this is the only recommended way of passing build-time secrets around.
* I was expecting to be able to write `GITHUB_PAT=$(cat /run/secrets/github_token) apt-get update...`, but couldn't get it to work in local tests without the `export`. If y'all reviewers see a way to simplify this, please shout.